src/LabelTrack.cpp

```diff
@@ -715,6 +731,13 @@ LabelFormat LabelTrack::FormatForFileName(const wxString & fileName)
 /// Import labels, handling files with or without end-times.
 void LabelTrack::Import(wxTextFile & in, LabelFormat format)
 {
+#ifdef EXPERIMENTAL_SUBRIP_LABEL_FORMATS
+   if (format == LabelFormat::WEBVTT) {
+      ::AudacityMessageBox( XO("Importing WebVTT files is not currently supported.") );
```

Comment:
This would be something to improve in a followup PR. I [see](https://developer.mozilla.org/en-US/docs/Web/API/WebVTT_API) that webVTT does some fancy stuff, so the importer would need to be told to strip away everything that we can't deal with, eg. 

```

WEBVTT

STYLE
::cue {
  background-image: linear-gradient(to bottom, dimgray, lightgray);
  color: papayawhip;
}

00:00:00.000 --> 00:00:04.000 position:10%,line-left align:left size:35%
Where did <b>he</b> go?
```

would need to be converted to

```
00:00:00.000 --> 00:00:04.000 
Where did <b>he</b> go?
```

and from there converted to a label. 

The tags (`<b>, <c>, <i>, <u>`, etc) inside the text can stay as they may be things you'd want to set or edit in label form, too - for example to add emphasis to certain words when a character does so too. The style would need to be discarded for the time being. 

A second followup would be to make a new exporter for the labels in a similar vein to the Export Audio exporter, which then could have custom fields for styles and such in case of vtt.